README.md

```diff
@@ -202,6 +202,17 @@ As a source author you do not have to worry about synchronization issues in case
 characters. asyncomplete.vim uses partial caching as well as ignores if the context changes when calling `asyncomplete#complete`.
 This is one of the core reason why the original context must be passed when calling `asyncomplete#complete`.
 
+### Suppress completion menu messages
```

Comment:
No need to add it in read me. I used it because we didn’t have docs yet.

Reply:
Are you sure about that? I think for now people look into README more then to doc, cause doc for this plugin doesn't have too much info.

Reply:
@pdavydov108 I would want to have docs in README as possible.
I need to remove the ones that are in there and move most of them to doc.